### PR TITLE
Add a page that lets us generate uncaught exceptions.

### PR DESCRIPTION
**Pull request by oponder**
To test the exception notification feature.

### CODE REVIEW

**Comment by oponder** [comment]
Deployed to ginger, I think exception notification isn't working.. digging into API logs now.

**Comment by oponder** [comment]
```
 Skipping exception notification, Slack Service is not defined.
```

```
{"caller":"github.com/giantswarm/api/service/exceptionnotification/creator/service.go:55","level":"debug","message":"Exception Notification: Request received","time":"2020-06-04T13:35:07.9892+00:00"}
{"caller":"github.com/giantswarm/api/service/exceptionnotification/creator/service.go:58","level":"debug","message":"Exception Notification: Skipping exception notification, Slack Service is not defined.","time":"2020-06-04T13:35:07.989245+00:00"}
```

File: src/components/ExceptionNotificationTest/ExceptionNotificationTest.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import { ErrorReporter } from 'lib/errors';
+import React, { ReactNode } from 'react';
+import Button from 'UI/Button';
+
+function generateUncaughtException() {
+  // @ts-ignore
+  // eslint-disable-next-line no-undef
+  return a_thing_that_doesnt_exist_on_purpose;
```
Comment (axbarsan) [inline src/components/ExceptionNotificationTest/ExceptionNotificationTest.tsx:8]: Is there any reason for not throwing a new error here? (e. g. `throw new Error("...")`)?

Comment (oponder) [reply src/components/ExceptionNotificationTest/ExceptionNotificationTest.tsx:8]: Oh, that makes more sense :)

**Pull request merged**